lib/listcoef.gd

```diff
@@ -101,26 +101,53 @@ DeclareOperation(
 #O  MultRowVector( <list>, <mul> )
 ##
 ##  <#GAPDoc Label="MultRowVector">
+##  <#GAPDoc Label="MultRowVectorLeft">
+##  <#GAPDoc Label="MultRowVectorRight">
 ##  <ManSection>
 ##  <Oper Name="MultRowVector" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Oper Name="MultRowVectorLeft" Arg='list1, [poss1, list2, poss2, ]mul'/>
+##  <Oper Name="MultRowVectorRight" Arg='list1, [poss1, list2, poss2, ]mul'/>
```

Comment:
You write later on that the 5-arg version will be removed. So, why document it here?

OK, I could understand if you want to document it for `MultRowVector`, for the sake of people who may have used the 5-arg version and want to find out what happened / is going to happen to it.

But in that case, I'd rather split this documentation: Write one mansection which just discusses `MultVectorLeft/Right`. In it, only list and document the 2-arg versions.

Then, have a *separate* man section for `MultRowVector`, which just says that its 2-arg version is just a synonym of `MultVectorLeft`, and that the 5-arg version is obsolete, should not be used, and is only there for backwards compatibility.

Bonus points for moving `MultRowVector` into `lib/obsolete.{gi,gd}`.